docs/changelog.rst

```diff
@@ -2,13 +2,16 @@
 Changelog
 =========
 
-* :bug:`1261` REST API now returns json error for invalid endpoints
-* :feature:`1230` Unless specifically provided gas price and gas limit are now dynamically calculated from the `eth_gasPrice()` and latest block's limit respectively.
+* :release:`0.3.0 <2018-02-21>`
+* :bug:`1217` Correctly decode network events in the REST API.
+* :bug:`1224` Fix internal server error on REST endpoint ``/events/tokens/`` for non-existing tokens.
+* :bug:`1261` REST API now returns json error for invalid endpoints.
+* :feature:`1230` Unless specifically provided gas price and gas limit are now dynamically calculated from the ``eth_gasPrice()`` and latest blocks limit respectively.
```

Comment:
this looks like the wrong issue number

Reply:
wasn't it #1473

Reply:
1473 was the internal sort of issue that explains to us what we want to do (iow use web3), while I guess 1230 was the first occurence of gas price being too low causing problems which eventually leads us to the solution.


But that's only a guess. This is almost a year ago.